openage/convert/entity_object/export/texture.py

```diff
@@ -135,6 +135,15 @@ def save(self, targetdir, filename, compression_level=1):
                                       - 3 = maximum possible compression; slowdown is 256x
         :type compression_level: int
         """
+        from ...service.export.png import png_create
+
+        COMPRESSION_LEVELS = {
+            0: png_create.CompressionMethod.COMPR_NONE,
+            1: png_create.CompressionMethod.COMPR_DEFAULT,
+            2: png_create.CompressionMethod.COMPR_GREEDY,
+            3: png_create.CompressionMethod.COMPR_AGGRESSIVE,
+        }
```

Comment:
I'd rather see this at class level, but it would require to make top-level import of png_create.

I'm not that familiar with codebase yet, so I don't know if it is desired or not.

Reply:
We do not want Cython modules to be imported at top level. That can mess with the builds.

Reply:
Alright then, thanks for explanation.